ansible/configs/ocp4-workshop/env_vars.yml

```diff
@@ -84,6 +84,8 @@ HostedZoneId: Z3IHLWJZOU9SRT
 aws_region: us-east-1
 # The key that is used to
 key_name: "default_key_name"
+# Whether to wait for AWS to return the instance state
+aws_instance_wait_for_state: true
```

Comment:
This should be false

Reply:
aws_instance_wait_for_stop: false
starting should always be true

Reply:
@joelbirchler does that mean we should be using separate variables?